dracut/30ignition/module-setup.sh

```diff
@@ -33,7 +33,29 @@ install() {
         sgdisk \
         useradd \
         usermod \
-        wipefs
+        wipefs \
+        cryptsetup \
+        clevis \
+        clevis-decrypt \
+        clevis-decrypt-sss \
+        clevis-decrypt-tang \
+        clevis-decrypt-tpm2 \
+        clevis-encrypt-sss \
+        clevis-encrypt-tang \
+        clevis-encrypt-tpm2 \
+        clevis-luks-bind \
+        clevis-luks-common-functions \
+        clevis-luks-unlock \
+        pwmake \
+        tpm2_create \
```

Comment:
How about we split these out as a separate `inst_multiple` with an explicit comment like:

```
# Needed for Clevis binding; note all binaries related to
# unlocking are included by the Clevis dracut modules.
```

?

Reply:
I don't mind splitting it. I do think it makes sense to keep them here though if we aren't explicitly including the clevis dracut modules in ignition-dracut (we're doing it in f-c-c).